src/components/playlistModal/playlistModal.js

```diff
@@ -2,15 +2,17 @@ import React, { useState } from "react";
 import { usePlaylistVideoContext } from "../../context/playlist/playlistContext";
 import { useSingleVideo } from "../../context/singleVideoContext/singleVideoContext";
 import { useToast } from "../../context/toastContext/toastContext";
+import { useWatchLaterCheckbox } from "../../context/watchLaterContext/watchLaterContext";
 import {
   addVideoToPlaylistApi,
   createPlaylistApi,
   deleteVideoFromPlaylistApi,
 } from "../../util/playlistApiCall";
+import { addToWatchLater,  removeFromWatchLater } from "../../util/watchLaterApi";
 import { Toast } from "../allComponents";
 import "./playlistModal.css";
-
 function PlaylistModal() {
+  const {videosInPlaylist,setwatchLaterCheckbox,setWatchLaterCheckboxDisptach}=useWatchLaterCheckbox(); 
   const {setToast,toastState,toastDispatch}=useToast();
   const {playlistNameValue,setPlaylistNameValue}=usePlaylistVideoContext();
```

Comment:
you can destructure properties of line 17, 18 in one line as they are coming from ` usePlaylistVideoContext`